default.nix

```diff
@@ -80,10 +80,13 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     patchShebangs kernel/
     patchShebangs dev/tools/
+    cat /proc/{cpu,mem}info || true
```

Comment:
I don't like having this here, it isn't related to the actual build. Please move it to `.gitlab-ci.yml`